Tryouts/Visuals/Avalonia/VisualUtils/WindowBasedNativeHost.cs

```diff
@@ -0,0 +1,25 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using System;
+
+namespace MorganStanley.ComposeUI.Tryouts.Visuals.Avalonia.VisualUtils
```

Comment:
'Visuals' link within the namespace specifies that it has dependencies on Visual packages - Avalonia or WPF. It separates it from Core (which does not depend on any visual packages). 
'Avalonia' link specifies that it is build on top of Avalonia - I expect that there might be also projects starting with `MorganStanley.ComposeUI.Tryouts.Visuals.WPF.`. 
The final link - 'VisualUtils' mean that these are the generic visual utilities, can be used by any project within ComposeUI. We shall probably also have some common Views under Avalonia, containing more specific code that applies only to that view. So I assume that we shall have e.g. project MorganStanley.ComposeUI.Tryouts.Visuals.Avalonia.AuthenticationView. 

If later it turns out that some of the links are superfluous, we can always go back and delete them, also rearranging the code correspondingly - but that would be a large architectural Jira on its own.

Reply:
Yep, my point is exactly that everything under Visuals should be related to visual packages, so I don't expect non-visual utils there hence '*Visual*Utils' is redundant, it could be simply 'Utils'. But as I said it's very minor.

Reply:
Oh, so I misunderstood what you've said. I think it is still better to have VisualUtils not to mix them with non-visual Utilities from the core :)

Reply:
You can't mix them since one is from .Core and the other is from .Visuals

Reply:
In my experience the last link in the namespace is pretty important so I believe that VisualUtils is better since they leave no doubt to the developers when it is from. I am not averse to change the namespace, but let us do it in a different Jira after the team discussion.